src/server/saved_objects/service/lib/search_dsl/query_params.js

```diff
@@ -98,9 +98,10 @@ function getClauseForType(schema, namespace, type) {
  *  @param  {(string|Array<string>)} type
  *  @param  {String} search
  *  @param  {Array<string>} searchFields
+ *  @param  {String} searchOperator
  *  @return {Object}
  */
-export function getQueryParams(mappings, schema, namespace, type, search, searchFields) {
+export function getQueryParams(mappings, schema, namespace, type, search, searchFields, searchOperator = 'OR') {
```

Comment:
I think we should move the default for `searchOperator` to the repository like we do for [other defaults](https://github.com/elastic/kibana/blob/381fa3c726a604f57f5fa0fdafe8b3700f93caa1/src/server/saved_objects/service/lib/repository.js#L301-L302)